README.md

```diff
@@ -406,6 +407,48 @@ It is possible to run gProfiler without using `perf` - this is useful where `per
 
 In this mode, gProfiler uses runtime-specific profilers only, and their results are concatenated (instead of scaled into the results collected by `perf`). This means that, although the results from different profilers are viewed on the same graph, they are not necessarily of the same scale: so you can compare the samples count of Java to Java, but not Java to Python.
 
+## Data format
+
+This section describes the data format output by the profiler. Some of it is relevant to the local output files (`.col`) files and some of it is relevant both for the local output files and for the data as viewable in the Performance Studio.
+
+### collapsed files
```

Comment:
Capitalize to header - `Collapsed Files"
It's also a convention in this README

Reply:
Applies to all new headings

Reply:
kk

Reply:
`attach-maps` I keep the same because that's the string in code